app/views/govuk_wrapper.scala.html

```diff
@@ -54,6 +58,14 @@
     <meta name="format-detection" content="telephone=no" />
 }
 
+@scriptElem = {
+    <script src="@routes.Assets.at("javascripts/scripts.js")" type="text/javascript"></script>
+    <script src="@routes.Assets.at("javascripts/accessibility_error_summary.js")" type="text/javascript"></script>
```

Comment:
Please remove the `accessibility_error_summary.js` since it's `direct-debit-frontend` specific and not related to timeout dialog.